process/block/alteredaccounts/errors.go

```diff
@@ -0,0 +1,15 @@
+package alteredaccounts
+
+import "errors"
+
+// ErrNilShardCoordinator signals that a nil shard coordinator has been provided
+var ErrNilShardCoordinator = errors.New("nil shard coordinator")
```

Comment:
Please do not duplicate these errors for each created package. This whole file can be deleted.
Use existing ones.

Reply:
We should definitely(work for another PR) move all high-frequently used errors in a separate package.